src/main/java/wedlog/address/logic/commands/GuestDeleteCommand.java

```diff
@@ -17,15 +24,45 @@ public class GuestDeleteCommand extends Command {
 
```

Comment:
Commenting here as I cannot comment directly on the above unchanged lines.
Let's add a "guest" keyword in front of the "delete" `COMMAND_WORD` to better reflect the usage of this command.